src/main/fulcro/client/primitives.cljc

```diff
@@ -2185,7 +2196,10 @@
       (p/queue! reconciler (into xs (remove symbol?) (keys v)))
       (when-not (empty? snds)
         (doseq [[remote _] snds]
+          (swap! (get-network-activity reconciler) assoc-in [remote :status] :active)
```

Comment:
Were these really necessary, or would this be fixed by including the remote in the networking code's call of `queue!`?

Reply:
@awkay Not sure this is the best place. Added it there to make the load markers `sync`,  since it seems to be the common function between load & mutations, before they go on the async queue. The end result I was trying to get is:
```
(df/load reconciler ...)
(-> reconciler prim/app-state deref ::net/status :remote) ; = :active
```

Reply:
OK, I see what you're going for.

Did it matter if the queue was with the remote or not?  Because you could just `conj` it onto the xs in the loop on the line before it.